src/test/java/rx/GroupByTests.java

```diff
@@ -90,4 +91,27 @@ public void call(String v) {
 
         System.out.println("**** finished");
     }
+    
+    @Test
+    public void groupsCompleteAsSoonAsMainCompletes() {
```

Comment:
Sorry, I pulled `master` branch instead of `1.x` (uh), confirm: it fails on 1.1.2.